planning/behavior_velocity_planner/src/scene_module/detection_area/debug.cpp

```diff
@@ -36,6 +36,14 @@ namespace
 {
 using DebugData = DetectionAreaModule::DebugData;
 
+struct MarkerColor
+{
+  float r;
+  float g;
+  float b;
+  float a;
+};
```

Comment:
It maybe better to use [this](http://docs.ros.org/en/noetic/api/std_msgs/html/msg/ColorRGBA.html)

Reply:
Committed the changes suggested to use the ROS message structure instead of custom struct. Please confirm if that is fine so I can resolve and rebase.